classes/class-wc-connect-help-provider.php

```diff
@@ -2,20 +2,263 @@
 
 if ( ! class_exists( 'WC_Connect_Help_Provider' ) ) {
 
+	define( 'WOOCOMMERCE_CONNECT_MINIMUM_WOOCOMMERCE_VERSION', '2.6' );
+	define( 'WOOCOMMERCE_CONNECT_MINIMUM_JETPACK_VERSION', '4.0.2' );
+	define( 'WOOCOMMERCE_CONNECT_MAXIMUM_SCHEMA_AGE_SECONDS', 3 * DAY_IN_SECONDS );
+
 	class WC_Connect_Help_Provider {
 
 		/**
+		 * @var WC_Connect_Service_Schemas_Store
+		 */
+		protected $service_schemas_store;
+
+		/**
 		 * @var WC_Connect_Service_Settings_Store
 		 */
 		protected $service_settings_store;
 
-		public function __construct( WC_Connect_Service_Settings_Store $service_settings_store ) {
+		/**
+		 * @array
+		 */
+		protected $fieldsets;
+
+		public function __construct( WC_Connect_Service_Schemas_Store $service_schemas_store, WC_Connect_Service_Settings_Store $service_settings_store ) {
 
+			$this->service_schemas_store = $service_schemas_store;
 			$this->service_settings_store = $service_settings_store;
+			$this->help_sections = array();
 
 			add_filter( 'woocommerce_admin_status_tabs', array( $this, 'status_tabs' ) );
 			add_action( 'woocommerce_admin_status_content_connect', array( $this, 'page' ) );
 
+			$this->add_fieldset(
+				'health',
+				_x( 'Health', 'This section displays the overall health of WooCommerce Connect and the things it depends on', 'woocommerce' ),
+				$this->get_health_items()
+			);
+
+			$this->add_fieldset(
+				'services',
+				__( 'Services', 'woocommerce' ),
+				$this->get_services_items()
+			);
+
+			$this->add_fieldset(
+				'debug',
+				__( 'Debug', 'woocommerce' ),
+				$this->get_debug_items()
+			);
+
+			$this->add_fieldset(
+				'support',
+				__( 'Support', 'woocommerce' ),
+				$this->get_support_items()
+			);
+
+		}
+
+		protected function get_health_items() {
+			$health_items = array();
+
+			// WooCommerce
+			// Only one of the following should present
+			// Check that WooCommerce is at least 2.6 or higher (feature-plugin only)
+			// Check that WooCommerce base_country is set
+			$base_country = WC()->countries->get_base_country();
+			if ( version_compare( WC()->version, WOOCOMMERCE_CONNECT_MINIMUM_WOOCOMMERCE_VERSION, "<" ) ) {
+				$health_item = $this->build_indicator(
+					'woocommerce_indicator',
+					'notice',
+					'indicator-error',
+					sprintf(
+						__( 'WooCommerce %s or higher is required (You are running %s)', 'woocommerce' ),
+						WOOCOMMERCE_CONNECT_MINIMUM_WOOCOMMERCE_VERSION,
+						WC()->version
+					),
+					''
+				);
+			} else if ( empty( $base_country ) ) {
+				$health_item = $this->build_indicator(
+					'woocommerce_indicator',
+					'notice',
+					'indicator-error',
+					__( 'Please set Base Location in WooCommerce Settings > General' ),
+					''
+				);
+			} else {
+				$health_item = $this->build_indicator(
+					'woocommerce_indicator',
+					'checkmark-circle',
+					'indicator-success',
+					sprintf(
+						__( 'WooCommerce %s is working correctly', 'woocommerce' ),
```

Comment:
this phrasing seems a bit weird to me. Maybe `WooCommerce is configured correctly` ?